seb_openedx/admin.py

```diff
@@ -1,7 +1,27 @@
 # -*- coding: utf-8 -*-
 """ Admin.py """
 from django.contrib import admin
+from django.db.models import TextField
+from seb_openedx.forms import SebCourseConfigurationForm
+from seb_openedx.models import (
+    ForbiddenCourseAccess,
+    SebCourseConfiguration,
+)
+from seb_openedx.widgets import ListWidget
+
+
+class SebCourseConfigurationAdmin(admin.ModelAdmin):
+    """Admin for the SebCourseConfiguration model."""
+    list_display = [
+        'course_id',
+        'user_banning_enabled',
```

Comment:
I would like to see something about the keys in the list_display view.
It does not have to be part of this PR, but it has to be included before this is final